pihole

```diff
@@ -144,6 +144,10 @@ Time:
 
   elif [[ "${1}" == "0" ]]; then
     # Disable Pi-hole
+    if grep -cq "BLOCKING_ENABLED=false" "${setupVars}"; then
```

Comment:
Do you think maybe we should `source` setupVars here and test the value rather than grepping the file?

Just thinking for the sake of consistency, like we do here

https://github.com/pi-hole/pi-hole/blob/a4ba8d4b8fd0e44e11deeecf33267c82c3d8735e/pihole#L308

Reply:
I know we source at different places. However, I also seem to recall that we said at some point that sourcing is dangerous as it could lead to executing arbitrary code - which may have entered `setupVars.conf` through an error - with elevated right. Hence, I chose a conservative approach that could be used as a blueprint for a redesign if anyone would want to do this.

Reply:
/shrug.

Either method works fine, tbh, I just didn't want to approve without first bringing it up...

Reply:
Good shout though. Both points are good. https://github.com/pruje/libbash.sh/blob/master/docs/libbash.md#lb_read_config may be the way to do this in the future, but for now. 

The grep approach though might need to account for a few different words for unset? `no` | `false` | `off` ? Is it possible for users to edit setupVars and put their own version of the setting?

Reply:
I guess they could, but anything other than `true` or `false` is going to be ignored in any case.

e.g if user sets `BLOCKING_ENABLED=unicorn` then it will be ignored!